pages/app-deploy/netcore/tips.js

```diff
@@ -110,6 +111,22 @@ https://aka.ms/dotnet-download`}
       </ul>
     </p>
 
+    <h3 id="timezone">تنظیم منطقه‌ی زمانی (TimeZone)</h3>
+    <p>
+      منطقه‌ی زمانی در برنامه‌های ASP.Net Core به‌طور پیش‌فرض بر روی Asia/Tehran تنظیم شده است اما شما می‌توانید منظقه‌ی زمانی برنامه‌ی خود را با استفاده از پارامتر <span className="code">timezone</span> در فایل <span className="code">liara.json</span> تغییر دهید:
+    </p>
+
+    <Highlight className="json">
+      {`{
+  "platform": "netcore",
+  "app": "dotnets-starter",
+  "port": 5000,
```

Comment:
Remove the port field. For .Net core apps, we automatically configure the right port.